blueocean-analytics-tools/src/main/resources/index.jelly

```diff
@@ -0,0 +1,4 @@
+<?jelly escape-by-default='true'?>
+<div>
+  This plugin is a part of Blue Ocean UI
```

Comment:
If this really is a plugin then this is the text that appears in the plugin manager, so it is important to have a clear description in this file.

Either way ... why not just update the file with a clear description?